awx/ui_next/src/components/PaginatedTable/ActionsTd.jsx

```diff
@@ -0,0 +1,33 @@
+import 'styled-components/macro';
+import React from 'react';
+import { Td } from '@patternfly/react-table';
+import styled, { css } from 'styled-components';
+
+// table cells will automatically grow beyond specified width to accomodate
+// multiple action buttons
+const ActionsGrid = styled.div`
+  display: grid;
+  grid-gap: 16px;
+  align-items: center;
+
+  ${props => {
+    const columns = '40px '.repeat(props.numActions || 1);
+    return css`
+      grid-template-columns: ${columns};
+    `;
+  }}
+`;
+
+export default function ActionsTd({ numActions = 1, children }) {
+  const width = numActions * 40;
```

Comment:
Could you automatically get the number of actions from the number of children?  If so, doing it that way would reduce the number of props being passed in.

Reply:
My initial thought was that doesn't work because some rows are missing certain actions... but now that I think about it again, the way it is now they still need to pass an empty div in the missing slots which isn't ideal...

I may need to put an index prop on the children so they can position themselves correctly in the grid

Reply:
OK, I reworked this to determine based on the number of children, and use ActionItems as children to render those children to null for actions/buttons where there user doesn't have RBAC permissions